File: example/scalalib/native/3-multi-module/bar/test/src/BarTests.scala
package bar

import utest.*
import scala.scalanative.unsafe.*

object BarTests extends TestSuite {
  def tests = Tests {
    test("simple one") {
      val result = HelloWorldBar.stringLength(c"hello")
      assert(result == 5)
      result
    }
  }
}

File: example/kotlinlib/basic/2-module-deps/bar/test/src/bar/BarTests.kt
package bar

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe

class BarTests :
    FunSpec({

        test("simple") {
            val result = generateHtml("hello")
            result shouldBe "<h1>hello</h1>"
        }

        test("escaping") {
            val result = generateHtml("<hello>")
            result shouldBe "<h1>&lt;hello&gt;</h1>"
        }
    })
